### PR TITLE
List all environment variables

### PR DESCRIPTION
**Pull request by evantahler**
A central place in the monorepo to list all of our environment variables and what they do.  This is mostly for us as developers to keep track of things. 

### Development

- [ ] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: documents/environment-variables.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- `DATABASE_URL` - URL to connect to Postgres/SQLite
+- `WEB_URL` - the FQDN that the Grouparoo application will resolve
+- `WEB_SERVER` - Should the web server be enabled?
+- `WORKERS` - How many background workers should this Grouparoo instance run (0 is off)
+- `SERVER_TOKEN` - A Unique and Random string that is used to identify other members of the cluster and secure SSR API requests.
+- `GROUPAROO_LOG_LEVEL` - What level should the application be logging at.
+- `GROUPAROO_LOGS_PATH` - where should logs be stored to disk?
+- `GROUPAROO_LOGS_STDOUT_DISABLE_COLOR` - disable colorizing STDOUT/STDERR logs
+- `GROUPAROO_LOGS_STDOUT_DISABLE_TIMESTAMP` - disable including the timestamp in log lines
+- `GROUPAROO_CONFIG_DIR` - Set the config directory path. Default is `${process.cwd()}/config`
```
Comment (pedroslopez) [inline documents/environment-variables.md:19]: Add `GROUPAROO_CONFIG_ARCHIVE`

File: documents/environment-variables.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+### `@grouparoo/dbt`
+
+- `DBT_PROFILES_DIR`
+
+### `@grouparoo/cloudwatch`
+
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_REGION`
```
Comment (pedroslopez) [inline documents/environment-variables.md:41]: Add `AWS_CLOUDWATCH_LOG_RETENTION_DAYS`

**Review by pedroslopez — COMMENTED** [review]
Added a couple that I noticed were missing. Also noting that depending on which gets merged first, #2695 will add another env var

**Comment by evantahler** [comment]
Thanks @pedroslopez I added the missing ENVs you found, and the new `GROUPAROO_EXPORT_LOG` too - assuming this merges in around the same time

**Review by pedroslopez — APPROVED** [review]

**Pull request merged**